src/software/logger/network_sink.cpp

```diff
@@ -21,12 +22,22 @@ NetworkSink::NetworkSink(unsigned int channel, const std::string& interface, int
 
 void NetworkSink::sendToNetwork(g3::LogMessageMover log_entry)
```

Comment:
Not sure if this matches the required signature for G3Logs `addSink`, but it's probably nice to have (I think the other sinks also copy it)
```suggestion
void NetworkSink::sendToNetwork(const g3::LogMessageMover& log_entry)
```